src/clients/notes_client.py

```diff
@@ -101,7 +101,7 @@ def create_notes(self, body: list[NoteModel]) -> LockstepResponse:
     page on the Lockstep Developer website. A note is a customizable 
     text string that can be attached to various account attributes 
     within Lockstep. You can use notes for internal communication, 
-    correspondence with clients, or personal reminders. The Note Model 
+    correspondence with clients_, or personal reminders. The Note Model 
```

Comment:
Why are we making this change?

Reply:
I see what happened. The I changed the original directory from clients to clients_. This would let me differentiate from the client's directory and the client's package. I see that it refactored the naming convention from the files to add an additional underscore. I can go ahead and refactor them back to not include the underscore.